### PR TITLE
fix: codesandbox not support slash import

### LINKED ISSUE #16514: List Component: Infinite & virtualized: This example on CodePen and CodeSandbox can not run
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Example code can not run.

### What does the proposed API look like?
Example code can run correctly.

### PR DESCRIPTION
**Pull request by ycjcl868**
Close #16514.

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

https://ant.design/components/select-cn/#components-select-demo-select-users

image

-----
[View rendered components/select/demo/select-users.md](https://github.com/ant-design/ant-design/blob/fix-select-demo/components/select/demo/select-users.md)
[View rendered components/transfer/demo/table-transfer.md](https://github.com/ant-design/ant-design/blob/fix-select-demo/components/transfer/demo/table-transfer.md)

### CODE REVIEW

File: components/select/demo/select-users.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ A complete multiple select sample with remote search, debounce fetch, ajax callb
 
 ```jsx
 import { Select, Spin } from 'antd';
-import debounce from 'lodash/debounce';
```
Comment (ztplz) [inline components/select/demo/select-users.md]: 为什么不改成 `import debounce from 'lodash.debounce'` 呢

Comment (ycjcl868) [reply components/select/demo/select-users.md]: 想了下，这样增加不少依赖，每使用一个就得加一个。

Comment (afc163) [reply components/select/demo/select-users.md]: `import debounce from 'lodash/debounce';` 体积小很多，推荐这么写。

Comment (afc163) [reply components/select/demo/select-users.md]: codesandbox 的问题要想个别的办法解决。

Comment (yesmeck) [reply components/select/demo/select-users.md]: 改进下这里 https://github.com/ant-design/ant-design/blob/master/site/theme/template/Content/Demo.jsx#L178-L187 找 dependencies 的逻辑爸

Comment (ycjcl868) [reply components/select/demo/select-users.md]: 有 lodash 的话，直接把 lodash external 出来

**Comment by zombieJ** [comment]
修好，这个 issue 可以一起处理了 #16514

**Comment by ycjcl868** [comment]
image

**Review by yesmeck — APPROVED** [review]

**Comment by afc163** [comment]
顺便看看 https://github.com/ant-design/ant-design/issues/16514

**Comment by ycjcl868** [comment]
https://deploy-preview-16509--ant-design.netlify.com/components/list-cn/#components-list-demo-infinite-virtualized-load


image

**Comment by ycjcl868** [comment]
发现了另一个问题，List 的加载无限会有空白

image

**Comment by zombieJ** [comment]
原本例子问题另起讨论，这个解决了就可以合了。

**Pull request merged**